system-tests-fixtures/default-schema/schema.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
looks the output of this schema is generated in `TestApi` folder? Can you please keep the folder name consistent i.e. either call the folder as `DefaultApi` or rename this folder to `test-schema`?